packages/types/validation/validationChatIdParam.ts

```diff
@@ -0,0 +1,7 @@
+import { string, object } from 'yup';
+
+export const validateChatIdParam = object({
+  chatId: string()
+    .matches(/^[a-z0-9]{24}$/i)
```

Comment:
object id matches can be done with a custom validator by mognoose
i think its called isValidObjectId